packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/take-profit.tsx

```diff
@@ -69,7 +69,11 @@ const TakeProfit = observer((props: TTakeProfitProps) => {
                 onChange={changeValue}
                 setCurrentFocus={setCurrentFocus}
                 tooltip_label={
-                    <Localize i18n_default_text='When your profit reaches or exceeds this amount, your trade will be closed automatically.' />
+                    is_accumulator ? (
+                        <Localize i18n_default_text="When your profit reaches or exceeds this amount, your trade will be closed automatically. Take profit can't be adjusted after your contract starts." />
```

Comment:
Part of text is duplicated (_When your profit reaches or exceeds this amount, your trade will be closed automatically_). If I move it into a constant, it might be not translated inside of `<Localize />`. Not sure, if there better way as I don't want to duplicate, please, share your ideas